source/content/debug-mysql-new-relic.md

```diff
@@ -48,9 +56,15 @@ Now that the problem has been found, it can be addressed. In this case, simply a
 ## Recap
 
 1. Use New Relic Pro to narrow and identify periods of time that have high load and/or slow response times.
+
 1. Narrow down the scope to one of those time periods and find the worst performing transactions.
+
 1. Within those transactions, go into the SQL trace to discover long running queries.
+
 1. Using SFTP, download the appropriate MySQL Slow Log to retrieve the query in its entirety.
+
 1. Connect to a safe MySQL server via CLI. Run the query to test the performance.
+
 1. If the query result is poor, use the `EXPLAIN` and `EXPLAIN EXTENDED` MySQL command to get additional information. You can also examine the MySQL tables for structural issues using `DESCRIBE` and `ANALYZE` commands.
+
 1. Once identified, fix the issue. This can be within the MySQL server itself if that's where the problem is, or it can be within the application by redoing code or configurations that are creating the errant query.
```

Comment:
`within MySQL server itself` implies server reconfiguration which is not feasible. Maybe `the SQL query`

Reply:
@marfillaster how does https://github.com/pantheon-systems/documentation/pull/6039 grab ya?

Reply:
Looks good